<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T2Class</class>
 <widget class="QMainWindow" name="T2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>T2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MouseEventHandler" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEditStart"/>
   <addaction name="actionEditStop"/>
   <addaction name="actionCalculate"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionEditStart">
   <property name="icon">
    <iconset resource="T2.qrc">
     <normaloff>:/T2/resource/开始编辑.png</normaloff>:/T2/resource/开始编辑.png</iconset>
   </property>
   <property name="text">
    <string>开始编辑</string>
   </property>
  </action>
  <action name="actionEditStop">
   <property name="icon">
    <iconset resource="T2.qrc">
     <normaloff>:/T2/resource/结束编辑.png</normaloff>:/T2/resource/结束编辑.png</iconset>
   </property>
   <property name="text">
    <string>结束编辑</string>
   </property>
  </action>
  <action name="actionCalculate">
   <property name="icon">
    <iconset resource="T2.qrc">
     <normaloff>:/T2/resource/凸包.png</normaloff>:/T2/resource/凸包.png</iconset>
   </property>
   <property name="text">
    <string>计算凸包</string>
   </property>
  </action>
  <action name="actionTxt">
   <property name="text">
    <string>添加数据</string>
   </property>
   <property name="toolTip">
    <string>添加数据</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MouseEventHandler</class>
   <extends>QGraphicsView</extends>
   <header>MouseEventHandler.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="T2.qrc"/>
 </resources>
 <connections/>
</ui>
